roles/job_runner/tasks/create_credential.yml

```diff
@@ -0,0 +1,257 @@
+- name: Define credentials, if error update AnsibleCredential status then end play
```

Comment:
We could potentially break out these into different playbooks called conditionally by main.yml.